### PR TITLE
Workflow

### PR DESCRIPTION
**Pull request by richsalz**
This uses GitHub's CI workflow to add "markdown lint" to all PR's.

It also creates two templates for reporting bugs or adding a crate.

I don't know how you maintain the site, @tarcieri, but we could integrate that into the CI workflow later if you want.

### CODE REVIEW

**Comment by tarcieri** [comment]
The site is just GitHub Pages, and in that regard there probably isn't much needed in the way of a workflow as a deploy will fail (leaving the previous deploy up) if something is wrong.

It really just boils down to these two files:

- https://github.com/The-DevX-Initiative/RCIG_Coordination_Repo/blob/main/_config.yml
- https://github.com/The-DevX-Initiative/RCIG_Coordination_Repo/blob/main/index.html

File: add_create.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (tarcieri) [inline add_create.md:1]: I assume you wanted to call this file `add_crate.md`?

**Review by tarcieri — CHANGES_REQUESTED** [review]
Looks like one of the filenames is wrong

File: add_create.md (hunk 1 above)
Comment (richsalz) [reply add_create.md:1]: damn my fingers have that typo welded-in.

**Review by tarcieri — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name Rust Crypto CI
```
Comment (tarcieri) [inline .github/workflows/ci.yml:1]: Oh crap, I just noticed this is missing a `:`.

Should be:

```yaml
name: Rust Crypto CI
```

Comment (tarcieri) [reply .github/workflows/ci.yml:1]: Fixed this in #45.

**Comment by tarcieri** [comment]
Hmmm, the workflows still aren't running: https://github.com/The-DevX-Initiative/RCIG_Coordination_Repo/actions/runs/1163506083

```
Error : .github#L1
No jobs defined in `jobs`
```

**Pull request merged**